### ISSUE TITLE
Null object reference error when trying to browse ExH galleries

### ISSUE THREAD

**Issue by Yandols** [NerdNumber9/TachiyomiEH#80]
**App version:** 8.2.0-EH and 8.4.1-EH.

**Android version:** 8.1

**Issue/Request:**

I am getting the following error when trying to browse ExHentai galleries. This does not occur when trying to browse galleries from other sources. I had this problem since 8.2.0-EH, upgrading to 8.4.1-EH does not seem to fix it.

```
Attempt to invoke virtual method 'org.jsoup.select.Elements org.jsoup.nodesElement.children()' on a null object reference
```

**Steps to reproduce (if applicable)**

 1. Go to Galleries/Catalogues
 2. Tap on Latest or Browse on ExHentai
 3. Error occurs with a retry button that does nothing

**Other details:**

* FoolSlide extension is installed.
* Other sources _(that I use)_ are not affected.
* I am still running on Ilwaz's fork because of this, which works perfectly fine for me, but is now behind on updates compared to your fork.

**Comment by NerdNumber9** [comment]
I believe this happens because ilwaz's fork changed the settings on your ExHentai account causing TachiyomiEH to become confused.

Try logging out and logging back into ExHentai in TachiyomiEH. This will cause the app to re-check the settings in your account and change anything that might be causing problems.

I'm going to close this issue now but if my above advice does not help, please re-open the issue and I'll take another look at this.

**Comment by Yandols** [comment]
Logging out and back in worked! But now a new problem has occured where no chapters are being loaded. 

The metadata of a gallery is downloaded but when I go to the chapters tab it just goes into an infinite loading. Pressing the "Play" FAB button just returns "Next chapter not found".

**Comment by NerdNumber9** [comment]
Are all galleries like that?

I think it's due to the recently added ability to view different versions of a gallery. On galleries with 100+ versions, loading the chapters screen can take well over a minute because the app has to search for all the older versions of a gallery.

**Comment by Yandols** [comment]
Apologies for the delayed response. I forgot about this after going back to using ilwaz's fork's app.

Anyways, I was browsing long on-going galleries so I did not realize that it was a problem relating to this. This does not happen on newly uploaded galleries however I did notice it flickered a few times before the 'clickable' chapter actually becoming available, it was not instant.

I read through the other related issues where you've implemented this feature, and I wonder if it's possible to add a toggle to turn this off and have the old functionality back? Or add a limit to the number parent galleries searched so that it won't take ~10 minutes to find a gallery that was updated/replaced over 500 times. While this is a really nice new functionality, it makes it a potential problem for galleries that have existed for a very long time and received so much updates. 

I might probably move this to it's own issue thread since I can't reopen this issue and it's not exactly related to the original issue anymore.
